ethstats/ethstats.go

```diff
@@ -68,7 +68,7 @@ const (
 	// connectionTimeout waits for the websocket connection to be established
 	connectionTimeout = 10
 	// delegateSendTimeout waits for the proxy to sign a message
-	delegateSendTimeout = 5
+	delegateSendTimeout = 10
```

Comment:
Curious if we've been seeing long response times for the delegate signing?

I thin `delegateSend` in this file isn't really clear what is happening, should this be renamed `delegateSignTimeout`?

Reply:
renamed, timeout back to 5 seconds

Reply:
renamed